core/domain/exp_domain_test.py

```diff
@@ -7076,10 +7076,11 @@ def test_all_html_strings_are_collected(self):
             'content_id': 'content',
             'html': '<p>Hello, this is state4</p>'
         }
-        state1.update_content(content1_dict)
-        state2.update_content(content2_dict)
-        state3.update_content(content3_dict)
-        state4.update_content(content4_dict)
+        to_domain = state_domain.SubtitledHtml.from_dict
```

Comment:
I think this place is also pending from @DubeySandeep's comment?

Reply:
updated.